### DISCUSSION TITLE
Đặt tên Adafruit feeed theo format

### DISCUSSION THREAD

**Discussion by khoavukhoa3004** [khoavukhoa3004/DADN_TTNT#11] [Q&A] [answered]
Tui nghĩ là: userName_room(2 chữ số xy)_sensorName_index(1 chữ số)
VD: khoa123_1_light_1

Ý nghĩa: 
-userName: cứ NMDK trc đã
- room:
+ x: tầng-0,1...
+ y: 0- khách, 1-ngủ, 2-bếp, 4-tắm
-sensorName: light, bulb....
-index: chỉ số sensor đó (phân biệt giữa 2 thiết bị giống nhau trong phòng)

**Comment by dangnguyenphuc** [comment, marked as answer]
Done
